packages/@okta/vuepress-site/docs/guides/registration-inline-hook/main/index.md

```diff
@@ -0,0 +1,155 @@
+
+> **Note:** Profile Enrollment and Registration Inline Hooks are only supported with the [Okta Sign-In Widget](/code/javascript/okta_sign-in_widget/) version 4.5 or later.
+
+To associate the Registration Inline Hook with your Profile Enrollment policy:
+
+1. In the Admin Console, go to **Security > Profile Enrollment**.
+
+1. Click the Pencil icon to edit the policy and associate it with your Registration Inline Hook.
+
+1. In **Enrollment Settings**, click the More Options icon and then select **Edit**. Select **Allowed** for **Sign-up** in the **For new users** section.
+
+1. Select your hook from the drop-down menu for **Use the following inline hook** under the options for **For new users** that you've set up an activated previously ("Guide Registration Hook Code"). The drop-down menu displays all the Registration Inline Hooks you have created.
+
+1. Click **Save**.
+
+Your Registration Inline Hook is now configured for Profile Enrollment. You are now ready to preview and test the example.
+
+> **Note:** Only one Inline Hook can be associated with your Profile Enrollment policy at a time.
+
+#### Enable the Registration Inline Hook in the Classic Engine
+
+<ApiLifecycle access="ea" />
+If you have a Classic Engine org, you must enable [self-service registration (SSR)](/docs/guides/set-up-self-service-registration/) to implement a Registration Inline Hook.
+
+> **Note:** Self-service registration and Registration Inline Hooks are only supported with the [Okta Sign-In Widget](/code/javascript/okta_sign-in_widget/) version 2.9 or later.
+
+To enable the Registration Inline Hook on the self-service registration page:
+
+1. In the Admin Console, go to **Directory** > **Self-Service Registration**.
+
+1. Click **Edit**.
+
+1. From the **Extension** field drop-down menu, select the hook you've set up and activated previously ("Guide Registration Hook Code").
+
+1. Click **Save**.
+
+The Registration Inline Hook is now enabled for self-service registration. You are now ready to preview and test the example.
+
+## Preview and test
+
+The external service example is now ready with code to receive and respond to an Okta call. The Okta org is now set up to call the external service using a Registration Inline Hook.
+
+In your Okta org, you can preview the request and response JSON right from the Admin Console. You can also test the code directly with self-registering users.
+
+### Preview
+
+To preview the Registration Inline Hook:
+
+1. In the Admin Console, go to **Workflow** > **Inline Hooks**.
+2. Select the Registration Inline Hook name (in this example, "Guide Registration Hook Code").
+3. Click the **Preview** tab.
+4. Select a user from your org in the first block titled "Configure Inline Hook request"; that is, a value for the  `data.userProfile` value.
```

Comment:
wouldn't this be `data.userProfile` **object**??

Also - there's an extra space in front of `data.userProfile`

Reply:
Yes! Good catch. Thank you.